gestalt-module/src/main/java/org/terasology/gestalt/module/dependencyresolution/DependencyInfo.java

```diff
@@ -109,25 +108,10 @@ public void setOptional(boolean optional) {
     }
 
     /**
-     * @return The range of supported versions
+     * Returns a predicate that yields true when applied to version that is within the version range described by this dependency information
      */
-    public VersionRange versionRange() {
```

Comment:
@keturn this is actually used in the Terasology modules builds :see_no_evil: Guess we need to bring this method back to make that work again.